bitrepository-core/src/test/java/org/bitrepository/protocol/performancetest/MessageBusNumberOfListenersStressTest.java

```diff
@@ -39,9 +39,13 @@
 import org.bitrepository.protocol.security.SecurityManager;
 import org.bitrepository.settings.repositorysettings.MessageBusConfiguration;
 import org.jaccept.structure.ExtendedTestCase;
```

Comment:
JAccept relies on TestNG. I got no idea whether we can get rid of both at the same time, or how great of an effort it would require. Ideally I think we would want to.